Assets/Scripts/PlayerController.cs

```diff
@@ -196,6 +207,16 @@ private void GetInput()
         {
             GameManager.Instance.Respawn();
         }
+
+        if (Input.GetKey(KeyCode.S) && PlayerState == PlayerStateEnum.Air && DistanceToGround() >= 0.5f)
+        {
+            PlayerState = PlayerStateEnum.Down;
+        }
+
+        if (Input.GetKey(KeyCode.S) && PlayerState == PlayerStateEnum.Air && DistanceToGround() >= 0.5f)
+        {
+            PlayerState = PlayerStateEnum.Down;
+        }
```

Comment:
Is it just me, or does this code do the same as the immediately preceding if block? If so, we could probably omit this